guide/qceff_probit.rst

```diff
@@ -0,0 +1,152 @@
+.. _QCEFF:
+
+Quantile-Conserving Ensemble Filter Framework
+==============================================
+
+The Quantile-Conserving Ensemble Filter Framework (QCEFF) tools are in the alpha release stage. 
+The DART development team (dart@ucar.edu) would be happy to hear about your experiences 
+and is anxious to build scientific collaborations using these new capabilities.
+
+The QCEFF options are set using a :ref:`qceff table <qceff table>` given as a namelist option to &algorithm_info_nml.
+
+   .. code-block:: text
+
+      &algorithm_info_nml
+         qceff_table_filename = 'qceff_table.csv'
+
+
+.. _QCEFF options:
+
+QCEFF options
+--------------
+
+QCEFF options are per quantity. For a given quantity, you specify the following 
+options as columns of the qceff_table:
+
+* Observation error information
+
+   Used to compute sample for this observation when using perfect_model_obs
+   to generate noisy observations.
+   
+     * bounded_below (default .false.) 
+     * bounded_above (default .false.)
+     * lower_bound   
+     * upper_bound
+
+
+* Probit distribution information 
+
+   Used in the computation of the probit transform.
+   The values needed are the bounds and the distribution type.
+   These can be different for all three cases (inflation, state, and extended_state priors)
+   
+     * distribution (one of :ref:`Distributions`)
+     * bounded_below (default .false.)
+     * bounded_above (default .false.)
+     * lower_bound
+     * upper_bound
+
+
+* Observation increment information
+
+     * filter_kind (one of :ref:`Filter kinds`)
+     * bounded_below (default .false.)
+     * bounded_above (default .false.)
+     * lower_bound
+     * upper_bound
+
+
+
+.. _qceff table:
+
+Creating a qceff table
+-----------------------
+
+The table has two headers, row 1 and 2.
+The first row is the version number.  The second row describes the :ref:`QCEFF options` corresponding to each column of the table. 
+These two headers must be present in your qceff table. 
+The :ref:`qcf trunc table` below shows the table headers, 
+and an example quantity QTY_TRACER_CONCENTRATION for the first 5 columns of the table. 
+There is a complete table with all 25 columns in `Google Sheets <https://docs.google.com/spreadsheets/d/1CRGHWc7boQt81pw2pDxEFY6WPyQeCh64OwPyoVMqijE/edit?usp=sharing>`_. You can copy and edit this table as needed.
```

Comment:
@mjs2369 I changed QTY_NAME: to QTY_EXAMPLE to address Ben's comment. 
![Screen Shot 2024-01-05 at 9 21 56 AM](https://github.com/NCAR/DART/assets/20047007/07589029-e85b-4722-a1d6-084b51c07491)

If this isn't sufficient to show that this is an example of a qty change it to whatever is appropriate.

Reply:
It does not matter - the header line needs to be there.

Reply:
I think the change to QTY_EXAMPLE is good.

I think what Brett is asking is if we should make all of the example CSVs in the lorenz_96_tracer_advection/work dir consistent in their headers in cell 2A, so we don't confuse people. We should make them all match the newest qcf_table_template (v1), which uses QTY_NAME instead of QTY_TEMPLATE. So all_bnrhf_qceff_table.csv, all_eakf_qceff_table.csv, and neg_qceff_table.csv would all get edited. 

We also need to to update the format when logging the qceff_table to fit QTY_NAME:
[https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L581](https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L581)

The new formatting will be:
`write(log_msg,'(A9, 2A14, 2A12, 3(A10, 2A14, 2A12), A12, 2A14, 2A12)') header2(:)`

Reply:
Also I believe that we should change the name of the Google Sheet template from qcf_table_template to qceff_table_template

And cell A1 should be changed to "QCEFF table version: 1"

This would require us to change the log formatting for the first header here as well:
[https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L578](https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L578)
to 
`write(log_msg, '(A6, A6, A9, A)') header1(:)`

And update the headers for all of the .csv files in lorenz_96_tracer_advection/work to "QCEFF table version: 1"

Reply:
ok @mjs2369  can you go ahead and make the changes to the google sheet, contents and name, and change the example .csv files in lorenz_96_tracer.